integration/mesos/src/main/java/alluxio/mesos/AlluxioFramework.java

```diff
@@ -72,6 +72,14 @@ public void run() {
       frameworkInfo.setPrincipal(Configuration.get(PropertyKey.INTEGRATION_MESOS_PRINCIPAL));
     }
 
+    // Publish WebUI url to mesos master.
+    String masterHostName = Configuration.containsKey(PropertyKey.MASTER_WEB_HOSTNAME)
```

Comment:
We have a utility get the master connect address from the configuration:
```java
NetworkAddressUtils.getConnectAddress(ServiceType.MASTER_WEB).toString()
```

Reply:
Thanks @aaudiber.